README.md

```diff
@@ -84,7 +84,12 @@ Tests are an important part of the development and evolution of this project,
 and new contributions are expected to include tests for any functionality
 change.  To run the tests, pass the `test` verb to the `bootstrap` script:
 
-    ./Utilities/bootstrap --build-tests test
```

Comment:
Can do! It definitely took me a while to figure out how to test SwiftPM (the test invocation in the Swift build script isn't straightforward either), and I think more docs could help.